pyccl/ccl_core.i

```diff
@@ -4,7 +4,7 @@
 /* put additional #include here */
 %}
 
-// SWIG black magic. Change the behaviour of setting A_SPLINE_MAX to thowing an
+// SWIG black magic. Change the behaviour of setting A_SPLINE_MAX to throwing an
 // error.
 %typemap(memberin) double A_SPLINE_MAX {
```

Comment:
Can't we remove that now and disallow changing `A_SPLINE_MAX` in the `__setattribute__` method?

Reply:
Yes we can, but I didn't want to touch the C-code for that one in case someone changes the Python-code in the future and forgets to reinstate that check. There are a couple of places this check appears in the C-code.